src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -806,6 +820,7 @@ const SshKeysRow = ({
                         testdata={keyInvalid ? "key-invalid" : undefined}
                         fieldId='public-key'>
                         <TextArea value={item.value || ""}
+                            autoFocus
```

Comment:
Is this wanted? I think there are multiple form fields in this modal no?

Reply:
The `autoFocus` makes it so that newly added text areas get the focus, and typing will continue in them. The test relies on that, since it doesn't simulate a paste operation, but rather a fast typist.

Reply:
But eslint doesn't want us to use it: "The autoFocus prop should not be used, as it can reduce usability and accessibility for users". Hmm.

Reply:
I recall we exclude this check somewhere else:


[jelle@t14s][~/projects/cockpit/cockpit-defer-lib-spawn]%git grep autoFocus
pkg/shell/superuser.jsx:                        autoFocus // eslint-disable-line jsx-a11y/no-autofocus